docs/plugins/plugin-script.asciidoc

```diff
@@ -64,13 +64,35 @@ sudo bin/elasticsearch-plugin install [url] <1>
 -----------------------------------
 <1> must be a valid URL, the plugin name is determined from its descriptor.
 
-For instance, to install a plugin from your local file system, you could run:
+==== Unix File System
```

Comment:
I think having a file that has three levels of titles in it upsets asciidoc. Check before merging please.